packages/taquito-michelson-encoder/src/tokens/list.ts

```diff
@@ -67,6 +67,10 @@ export class ListToken extends Token {
   }
 
   public ExtractSchema() {
-    return ListToken.prim;
+    const valueSchema = this.createToken(this.val.args[0], this.idx);
+    return {
```

Comment:
The main purpose of MichelsonEncoder is to abstract Michelson language and "prim" is really something proper to the Michelson language. Thus, I don't think we should return a "prim" in the schema.
An alternative could be:
```
return {
ListToken.prim: valueSchema.ExtractSchema()
}
```

Reply:
This has been fixed.